src/Pos/Ssc/GodTossing/Toss/Trans.hs

```diff
@@ -89,8 +90,8 @@ instance MonadTossRead m =>
     getCommitments = TossT $ (<>) <$> use tmCommitments <*> getCommitments
     getOpenings = TossT $ (<>) <$> use tmOpenings <*> getOpenings
     getShares = TossT $ (<>) <$> use tmShares <*> getShares
-    getVssCertificates =
-        TossT $ (<>) <$> use tmCertificates <*> lift getVssCertificates
+    getVssCertificates = certs <$> getVssCertData
```

Comment:
It was wrong change. You should consider `tmCertificates` in `TossT`. It was fixed in #1160.